packages/gatsby-admin/gatsby-config.js

```diff
@@ -2,7 +2,10 @@ module.exports = {
   plugins: [
     {
       resolve: "gatsby-plugin-react-helmet",
-      options: {},
+      options: {
+        test: true,
+        bla: false,
```

Comment:
`bla` is removed in #26916

but i don't see if the `test` option used? is this a test leftover?

Reply:
Yea, this is just me playing around with configuring plugins in Admin!